Add barrier 6 (takes time) to barriers.md in reproducibility chapter (#704)
alan-turing-institute/the-turing-way: book/content/reproducibility/04/barriers.md

> ## Barrier 6

Reviewer: This formats the barrier itself as the heading (rather than having the heading say "Barrier 2" and then reading down to know what it is. I think it will make the chapter easier to scan if someone's quickly reading through.
```suggestion
## Takes Time
```

> Takes Time

Reviewer: Move barrier heading to sub-title
```suggestion
```

> The time needed to maintain a reproducible project presents a significant barrier. At the start of the project considerable time needs to be invested in designing and setting up the reproducibility pipepine. This may include a testing framework , continuous integration, Github repository and data rights. Time may also be spent communicating with collaborators to agree which parts of the project may be open source and how they are shared, this may include creating synthetic data.Throughout the project, considerable time needs to be spent mainting the reprodicibility pipeline including refactoring, testing and resolving conflicts. At project conclusion time usually spent modifying analysis may be saved thanks to the reproducibility pipeline. However, time may be lost at review due to framework explanation and suggested adaptations.

Reviewer: ```suggestion
Making an analysis reproducible takes time and effort, particularly at the start of the project.
This may include agreeing upon a [testing framework](.../../testing/testing), setting up [version control](../../version_control) such as a Github repository and [continuous integration](../../continuous_integration/continuous_integration), and [managing data](../../rdm/rdm.html).
Throughout the project, time may be required to maintain the reproducible pipeline.

Time may also be spent communicating with collaborators to agree which parts of the project may be open source and when and how these outputs are shared.
However, time may be lost at review due to framework explanation and suggested adaptations.

There will be fewer errors and less time spent correcting mistakes.
The analysis pipeline can be easily adapted as needed in response to co-author and reviewer requests.
```
